src/Tizen.NUI/src/public/ParticleSystem/ParticleEmitter.cs

```diff
@@ -0,0 +1,575 @@
+/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.ComponentModel;
+
+namespace Tizen.NUI.ParticleSystem
+{
+    using Tizen.NUI.BaseComponents;
+
+    /// <summary>
+    /// Enum defining blending options when rendering the particles.
+    /// </summary>
+    public enum ParticleBlendingMode
+    {
+        Additive = 0,
+        Screen = 1,
+        Default = Additive
+    }
+    
+    /// <summary>
+    /// Internal class defining data types stored in the data streams
+    /// </summary>
+    internal enum StreamType
+    {
+        Float = 0,
+        FloatVector2 = 1,
+        FloatVector3 = 2,
+        FloatVector4 = 3,
+        Integer = 4,
+        IntVector2 = 5,
+        IntVector3 = 6,
+        IntVector4 = 7,
+    }
+    
+    /// <summary>
+    /// Class ParticleEmitter creates a single emitter attached to a specified
+    /// View. ParticleEmitter is responsible for spawning and updating particles.
+    ///
+    /// Emitter must contain:
+    /// ParticleSource - responsible for spawning new particles
+    /// ParticleModifier(s) - responsible for updating particles in the system
+    ///
+    /// ParticleSource and ParticleModifier callback interfaces should not be accessing
+    /// Event side (NUI) objects. Both callbacks are executed on Update thread.
+    /// </summary>
+    public class ParticleEmitter : BaseHandle
```

Comment:
```suggestion
    [EditorBrowsable(EditorBrowsableState.Never)]
    public class ParticleEmitter : BaseHandle
```

Please make this class as hidden, if you don't prepare ACR.